assets/queries/terraform/alicloud/ram_password_security_settings_require_password_expire_after_number_days/metadata.json

```
@@ -0,0 +1,11 @@
+{
+    "id": "2bb13841-7575-439e-8e0a-cccd9ede2fa8",
+    "queryName": "Ram Account Password Policy Not Required Password Expire after set of days",
```

```suggestion
    "queryName": "Ram Account Password Policy Max Password Age Unrecommended",
```
Please, also update the query folder name to `ram_account_password_policy_max_password_age_unrecommended` and update the positive_expected_value.json